app/models/pacbio/library.rb

```diff
@@ -93,6 +93,23 @@ def sequencing_plates
       wells&.collect(&:plate)
     end
 
+    # Return the data to be published as the source of an aliquot
```

Comment:
Same again. It does feel like publish_data should be a single object / class / hash.